tests/models/owlvit/test_modeling_owlvit.py

```diff
@@ -677,52 +676,6 @@ def _create_and_check_torchscript(self, config, inputs_dict):
 
             self.assertTrue(models_equal)
 
-    def test_model_outputs_equivalence(self):
```

Comment:
This test should not be removed.

Reply:
@sgugger It is not removed, it is a common test that was previously overwritten due to selectively computing unmodified and modified base model embeddings, the test still passes.